_posts/2018-11-26-resize-text.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Quick test: Resize text"
+description: "How to test the Resize Text WCAG Success Criterion"
+author: luis_garcia
+published: true
+categories:
+  - Quick Tests
+---
+
+Resize text and make sure everything on the page works. Repeat until you get to 200% zoom. Seems simple? It is.
+
+## Essential Abilities
+
+* **Sight**: This method tests the visual appearance of content on the page
+* **Interact with content via a pointer**: This method tests mouse interactions
+
+If you know of a method that doesn't require either of these abilities, please update this Quick Test.
+
+## Background
+
+User's with low vision may need to increase the size of text in order to use the page. 
+
+When the user resizes text, any of the following can break the page:
+
+* text on the page is no longer readable
+* content is lost or truncated
+* functionality or inter
+
+## How To Test
+
```

Comment:
could we add a quick sentence or two here so as to not go from one heading directly into another?